Makefile

```diff
@@ -29,3 +29,7 @@ build: test
 # image will build a docker image
 image:
 	docker build -t prebid-server .
+
+# format runs format
+format:
+	./scripts/format.sh -f true
```

Comment:
this can be just `./scripts/format.sh`, because format is set to true at the beginning of the script

Reply:
I don't see anywhere in the script where it access command line arguments. As far as I can tell, the `-f true` does nothing, nor would any other command line arguments. Unless I am missing something, and there are arguments that would change its behavior, we should omit the command line arguments as it implies that there is some command line functionality.

Reply:
In format.sh script we read a value from `-f`:
```bash

AUTOFMT=true
while getopts 'f:' OPTION; do
  case "$OPTION" in
    f)
      AUTOFMT="$OPTARG"
      ;;
  esac
done
```
and set it to AUTOFMT

this variable can be passed from validate.sh if it is executed with `--nofmt`.

Reply:
Ahh, okay, I missed that. Thanks.